README.md

```diff
@@ -73,7 +86,7 @@ npm run firebase login
 npm run firebase use staging
 
 # Copy the runtime config locally
-npm run --silent firebase functions:config:get > .runtimeconfig.json
+npm run --silent firebase functions:config:get > ./functions/.runtimeconfig.json
```

Comment:
Nice catch!

Seems we missed updating that when we added the `cd ..` part in https://github.com/sparkletown/sparkle/pull/1303